### ISSUE TITLE
Add audio cues for Go to Next/ Previous Change commands

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#166518]
When navigating changes with f5 or the `Go to Next` / `Go to Previous` commands, we should play audio cues - modified, inserted, deleted, based on the decorations like we now do for the `DiffReview`

This was requested here https://github.com/microsoft/vscode/issues/147226 

cc @isidorn for thoughts

**Comment by isidorn** [comment]
I think this makes good sense.

**Comment by meganrogge** [comment]
@jooyoungseo this is now in insider's

**Comment by jooyoungseo** [comment]
@meganrogge -- Doesn't work. Alt+F5/Alt+Shift+F5 jumps to the next/previous changes with no audio cues being played. Screen reader does not announce focused line, either. Tested with JAWS and NVDA on Windows 11.

Version: 1.75.0-insider (system setup)
Commit: 6d40104789d03d41b8866a1e57847dae14c5cf0d
Date: 2023-01-12T05:34:21.597Z
Electron: 19.1.9
Chromium: 102.0.5005.194
Node.js: 16.14.2
V8: 10.2.154.23-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes

**Comment by meganrogge** [comment]
@jooyoungseo I tested on macOS only. I'll try windows, though it should work the same.

**Comment by meganrogge** [comment]
The audio cues do work for me on Windows - can you pls check your settings? 

The selection of the line is happening, but it seems that doesn't impact the screen reader on Windows in the same way it does on mac.

**Comment by isidorn** [comment]
@meganrogge I just tired with latest VS Code Insiders on macOS.
I set "audioCue.diffInserted" to "on" and I opened a regular file (not a diff editor) with some inserted lines. I navigated to the lines and no sound was played 🐛 

When I go to F1 > list audio cues > and navigate to the new diff sounds they nicely get played

**Comment by meganrogge** [comment]
As we already chatted about, this issue was specifically about when the `Go to next/ previous difference` are used. Support for when the cursor moves is tracked here 😄 https://github.com/microsoft/vscode/issues/171256
